lib/client/getLinkForRoadmapChild.ts

```diff
@@ -0,0 +1,38 @@
+import { paramsFromUrl } from '../paramsFromUrl';
+import { IssueData } from '../types';
+import { getCrumbStringFromIssueData } from '../breadcrumbs';
+import { ViewMode } from '../enums';
+
+interface GetLinkForRoadmapChildOptions {
+  issueData?: Pick<IssueData, 'html_url' | 'children' | 'parent'> & Partial<Pick<IssueData, 'children'>>;
+  query?: {crumbs?: string};
+  currentRoadmapRoot?: Pick<IssueData, 'html_url' | 'title'>;
+  viewMode?: ViewMode;
```

Comment:
Because we do not need a full issueData. We need an IssueData type that has the appropriate fields. It's not a deviation, it's a way to ensure type safety while being explicit about the properties you're actually using. There are a large number of benefits.